src/fabric_rpc2.erl

```diff
@@ -96,7 +96,14 @@ changes(DbName, #changes_args{} = Args, StartVector, DbOptions) ->
     changes(DbName, [Args], StartVector, DbOptions);
 changes(DbName, Options, StartVector, DbOptions) ->
     set_io_priority(DbName, DbOptions),
-    #changes_args{dir=Dir} = Args = lists:keyfind(changes_args, 1, Options),
+    #changes_args{dir=Dir, filter=Filter} = Args0 = lists:keyfind(changes_args, 1, Options),
+    Args = case Filter of
+        {fetch, Style, {DbArg, JsonReq, DDocId, FName}} ->
```

Comment:
I suppose it wouldn't hurt to add that in, but right now the only caller sends a binary. Do you think it's better to fail loudly with a pattern match error here, or silently skip the fetch command and load the slow way? I think I would honestly rather see this throw a 500 than fail silently back to the slow approach.